sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -905,6 +922,7 @@ class FileFormatChecks private (
       schema: StructType,
       fileType: FileFormatType,
       op: FileFormatOp): Unit = {
+    tagTimestampUtcCheck(meta, schema.fields)
     tagUnsupportedTypes(meta, sig, schema.fields,
```

Comment:
Checked twice: `tagTimestampUtcCheck`, `tagUnsupportedTypes`
Can we update `tagUnsupportedTypes` to return the error string?